test/e2e/cypress/e2e/checks_catalog.cy.js

```diff
@@ -68,8 +68,9 @@ context('Checks catalog', () => {
   describe('Individual checks data is expanded', () => {
     it('should expand check data when clicked', () => {
       cy.get('.check-panel').should('not.exist');
+      cy.get('div.check-row').should('have.length', 4);
```

Comment:
Having this `4` hardcoded would make the test fail if at some point we add a new checks group.
Could we check maybe if the groups are there some other way?
Maybe `.should('have.length.least', 1)`